### DISCUSSION TITLE
Error message when running the program

### DISCUSSION THREAD

**Discussion by HudsonLanier** [Nelson-Gon/pycite#10] [Q&A] [answered]
I was running the program per the instructions listed in README.rst and when I input line 173 or README.rst (my_citations = PyCite(et cetera...) the program worked correctly for a time and then gave the error message

Traceback (most recent call last):
  File "<input>", line 1, in <module>
  File "/home/hudsonlanier/.virtualenvs/pycite/lib/python3.9/site-packages/pycite/pycite.py", line 132, in cite
    title = bs4_link.find_all("h1", {"class": "content-title"})[0].text
IndexError: list index out of range

This error occurred when the program was trying to cite this article (https://www.sciencedirect.com/science/article/pii/S0301051117302314)

**Comment by Nelson-Gon** [comment]
Hello,

Thank you for raising this. I can confirm that I did experience this (earlier today) and pushed a fix that hopefully fixed it. Can you still reproduce this error with this version https://github.com/Nelson-Gon/pycite/tree/5dbe9743baad34b783820631c98a29ad2632f3fb?

Thanks,

Nelson

**Reply by HudsonLanier** [reply]
Negative. Having the same problem. I deleted PyCite off my computer and reinstalled it at the link you posted. Having the same problem with the same error message.

**Reply by Nelson-Gon** [reply]
Thanks, could you please start an issue at https://github.com/Nelson-Gon/pycite/issues with minimal steps to reproduce? I will take a look at it.

Keep Builiding,

Nelson

**Reply by HudsonLanier** [reply]
Cool, done.

**Comment by Nelson-Gon** [comment]
Hi, 

I just looked at the error message. It seems you changed the code somewhere and used the HTML tag "h1" (title = `bs4_link.find_all("h1", {"class": "content-title"})[0].text)` instead of a span as in the source: 
https://github.com/Nelson-Gon/pycite/blob/5dbe9743baad34b783820631c98a29ad2632f3fb/pycite/sciencedirect.py#L13

Since, this is a science direct article, it should call science direct methods (sd_*) which for the title use the above line.

Please let me know if this helps.

**Comment by Nelson-Gon** [comment, marked as answer]
Following this in #11.
